app/Contracts/Services/LoginServiceInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace App\Contracts\Services;
+use Illuminate\Http\Request;
+
+interface LoginServiceInterface{
```

Comment:
```suggestion
interface LoginServiceInterface {
```
Add EOF.
Even though you checked the PR check list No1 in excel sheet, you missed here.

Reply:
Removed